### PR TITLE
Ea/release 124 curb CA Cert fixes

### PR DESCRIPTION
**Pull request by eanders**
[//]: # 'remove this if PR is for a release-* branch'
# _Please squash merge this PR_

## Description

Hotfix for an alpine3.20/curb incompatibility.
https://green-river.sentry.io/issues/5634805716/?referrer=slack

## Type of change
[//]: # 'remove options that are not relevant'
- [x] Dependency update

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

**Comment by eanders** [comment]
Not sure why, but my editor insisted on fixing rubocop errors on save this time... so VariableImporter includes some additional changes.

**Review by ttoomey — APPROVED** [review]
Looks good, thank you!

I'll flag this in the container os issue since we can back this out if we aren't in alpine.

File: spec/models/curl_spec.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+RSpec.describe Curl, type: :model do
+  describe 'CA Certificate Works' do
+    it 'does not throw an error when curling over https' do
+      expect do
+        Curl::Easy.perform('https://services.etosoftware.com/') do |http|
+          # http.cacert = '/etc/ssl/certs/ca-certificates.crt'
+          http.verbose = true
+        end
+      end.to_not raise_error
```
Comment (awisema) [inline spec/models/curl_spec.rb:22]: I wonder if you should 'x' this test, and copy it and flip the sense around on this so this passes as long as the work around is required?

**Review by awisema — APPROVED** [review]

File: spec/models/curl_spec.rb (hunk 1 above)
Comment (eanders) [reply spec/models/curl_spec.rb:22]: Oh, this test is supposed to fail for right now.  I'm testing some other mechanisms (I'd really like to set the CA Path in an initializer if possible.)

**Pull request merged**